scripts/zones/Konschtat_Highlands/mobs/Ghillie_Dhu.lua

```diff
@@ -0,0 +1,61 @@
+-----------------------------------
+-- Area: Konschtat Highlands
+--  NM:  Ghillie Dhu
+-----------------------------------
+
+require("scripts/globals/utils");
+require("scripts/globals/status");
+require("scripts/globals/fieldsofvalor");
+
+-----------------------------------
+-- onMobInitialize
+-----------------------------------
+
+function onMobInitialize(mob)
+    -- Hits especially hard for his level, even by NM standards.
+    mob:addMod(MOD_ATT, 50); -- May need adjustment along with cmbDmgMult in mob_pools.sql
+end;
+
+-----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+end;
+
+-----------------------------------
+-- onMobRoam
+-----------------------------------
+
+function onMobRoam(mob)
+    -- Fairly sure he shouldn't be storing up max TP while idle.
+    if (mob:MOD_REGAIN() ~= 0) then

```

Comment:
Dammit, left wrong copy of file in my trunk. Amending..I did that earlier too so I'm fixing same line TWICE.